my-app/src/helpers/services.ts

```diff
@@ -0,0 +1,49 @@
+import axios from "../axios";
+import {BlogPostType} from "../pages/homePage/BlogHomeContext";
+
+type InputType = {
+    title: string
+    text: string
+    imageUrl: string
+}
+type RegisterInput = {
+    email: string,
+    password: string,
+    fullName: string,
+}
+type LoginInput = {
+    email: string,
+    password: string,
+}
+export const services = {
+    blog: {
+        list: async () => {
+            const response = axios.get('/posts').then(res => res.data)

```

Comment:
![Screenshot 2022-11-13 at 20 25 08](https://user-images.githubusercontent.com/38957865/201540396-c6d6cb0d-cf31-4a27-98df-2839562949e8.png)


That's how you initialise it and how you use it. Here I put everything in one place, so that you can easily see it, but you should create config only once for whole app, maybe it makes sense to move it to custom hook, since it will be the same for every api